### PR TITLE
Limit the number of binds per consumer that can run concurrently.

### PR DESCRIPTION
**Pull request by awood**
I plan on submitting this as a PR to master as well.

### CODE REVIEW

File: server/src/main/java/org/candlepin/pinsetter/tasks/EntitlerJob.java [hunk 1]
```diff
@@ -82,9 +89,33 @@ public static JobDetail bindByPool(String poolId, Consumer consumer, Integer qty
         JobDetail detail = newJob(EntitlerJob.class)
             .withIdentity("bind_by_pool_" + Util.generateUUID())
             .requestRecovery(false) // do not recover the job upon restarts
+            .storeDurably()
```
Comment (mstead) [inline server/src/main/java/org/candlepin/pinsetter/tasks/EntitlerJob.java]: Any particular reason why you had to set this? Just curious.

Comment (awood) [inline server/src/main/java/org/candlepin/pinsetter/tasks/EntitlerJob.java]: It's required for trigger-less jobs.  You get an error if it is not set.

**Comment by mstead** [comment]
retest this please

**Comment by mstead** [comment]
To make sure that I understand this PR, you are basically checking how many EntitlerJobs are running for an Owner, and if the threshold is reached, queue the job to be executed later?

**Comment by mstead** [comment]
@awood If my understanding of the PR above is correct, then this looks Ok to me.

Couple of things:
* please add some detail about the fix in the commit message
* Spec test failures seem to be due to Jenkins.

**Comment by mstead** [comment]
ACK.

**Pull request merged**